src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -624,7 +624,10 @@ suggestExtendImport exportsMap contents Diagnostic{_range=_range,..}
             Just result <- addBindingToImportList ident importLine
             = [("Add " <> renderIdentInfo ident <> " to the import list of " <> mod, [TextEdit range result])]
           | otherwise = []
-        lookupExportMap binding mod 
+        renderImport IdentInfo {parent, rendered}
```

Comment:
haha, it came back

Reply:
That was my fault, trying to fix a merge conflict